backend/capellacollab/projects/toolmodels/modelsources/git/gitlab/handler.py

```diff
@@ -39,10 +40,10 @@ async def get_project_id_by_git_url(self) -> str:
                 return (await response.json())["id"]
 
     async def get_last_job_run_id_for_git_model(
-        self,
-        job_name: str,
+        self, job_name: str, project_id: t.Optional[str] = None
```

Comment:
```suggestion
        self, job_name: str, project_id: str | None = None
```
And I think the import can then also be removed